apps/aesophia_http/src/aesophia_http_handler.erl

```diff
@@ -47,6 +47,7 @@ handle_request_json(Req0, State = #state{ validator = Validator,
             Req = cowboy_req:reply(400, #{}, Body, Req1),
             {stop, Req, State}
     catch error:_Error ->
+            io:format("Exception: ~p\n~p\n", [_Error, erlang:get_stacktrace()]),
```

Comment:
not OTP21 compatible, but fine

Reply:
Oh, that one wasn't supposed to go in...